src/util/sandbox/fd_sandbox_linux.c

```diff
@@ -0,0 +1,281 @@
+#ifdef FD_HAS_SANDBOX_LINUX
+
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
+#endif
+
+#include "fd_sandbox_linux_private.h"
+
+#include <errno.h>        /* errno */
+#include <fcntl.h>        /* open */
+#include <linux/audit.h>
+#include <linux/seccomp.h>
+#include <sched.h>        /* CLONE_*, setns, unshare */
+#include <stddef.h>
+#include <stdio.h>        /* snprintf */
+#include <stdlib.h>       /* clearenv, mkdtemp*/
+#include <sys/mount.h>    /* MS_*, MNT_*, mount, umount2 */
+#include <sys/prctl.h>
+#include <sys/resource.h> /* RLIMIT_*, rlimit, setrlimit */
+#include <sys/stat.h>     /* mkdir */
+#include <sys/syscall.h>  /* SYS_* */
+#include <unistd.h>       /* set*id, sysconf, close, chdir, rmdir syscall */
+
+#include "../log/fd_log.h"
+
+#define ARRAY_SIZE(arr) (sizeof(arr) / sizeof((arr)[0]))
+
+
+int  fd_sandbox_linux ( const fd_sandbox_profile_linux_t * profile );
+
+void
+fd_sandbox_profile_init_common( fd_sandbox_profile_linux_t * profile ) {
+  profile->initialized      = true;
+  profile->chroot_path      = NULL;
+  profile->netns            = "fd-frank-netless",
+  profile->seccomp_prog     = NULL;
+  profile->user             = "nobody";
```

Comment:
This also should be an anonymous user, not named or set up before hand. See chatgpt response for a question like "Can I create an anonymous user  in C code that isn't named, for use with a user namespace?"